contracts/SubscriptionBilling.sol

```diff
@@ -239,6 +239,11 @@ contract SubscriptionBilling is HasNoContracts {
             }
         }
 
+        // Make sure to check the last federation member as well.
```

Comment:
I don't understand why you didn't change the i in line 230 to i++ in order to achieve this...